src/homepageExperience/containers/HomepageContainer.tsx

```diff
@@ -54,7 +54,15 @@ export const HomepageContainer: FC = () => {
   const squareGridCardSize = '200px'
 
   // events handling
-  const logPythonEvent = () => {
+  const logGoWizardClick = () => {
+    event('firstMile.goWizard.clicked')
+  }
+
+  const logNodeWizardClick = () => {
+    event('firstMile.goWizard.clicked')
```

Comment:
looks good to me!  should this be `firstMile.goWizard.clicked`?

Reply:
Thanks! Copypasta fixed.